ivy/functional/frontends/tensorflow/raw_ops.py

```diff
@@ -17,11 +17,13 @@ def Acosh(*, x, name="Acosh"):
 
 
 def Add(*, x, y, name="Add"):
-    return ivy.add(x, y)
+    Add = tf_frontend.add
```

Comment:
no you can remove the entire function definition, and just write `Add = tf_frontend.add` as one line in this file, nothing else is needed :)